src/main/java/com/fcgl/madrid/points/service/UserPointsService.java

```diff
@@ -19,4 +33,37 @@ public UserPointsService(UserPointsRepository userPointsRepository) {
     public List<UserPoint> findAll() {
         return userPointsRepository.findAll();
     }
+
```

Comment:
Can we add some `Resilience4j` logic above this function please

Similar to this: https://github.com/fcgl/forum/pull/10/files#diff-40773a84dd74d64ebad9a4a76a122205R7, https://github.com/fcgl/forum/pull/10/files#diff-40773a84dd74d64ebad9a4a76a122205R122
```
@CircuitBreaker(name = "backendA", fallbackMethod = "fallback")
...
```

```
    /**
     *
     * @param ex Exception
     * @return ResponseEntity<InternalStatus>
     */
    private ResponseEntity<GetUserPointsResponse> fallback(Long userId, Exception ex) {
        String message = "Fallback: " + ex.getMessage();
        InternalStatus internalStatus = new InternalStatus(StatusCode.UNKNOWN, 500, message);
        GetUserPointsResponse postResponse = new GetUserPointsResponse(internalStatus, null);
        return new ResponseEntity<GetUserPointsResponse>(postResponse, HttpStatus.INTERNAL_SERVER_ERROR);
    }
```

Reply:
Can we also change the response type to be GetUserPointsResponse

That will be a class with these two attributes: UserPoint, InternalStatus

You can use this as reference